test/test_support.ts

```diff
@@ -178,12 +178,15 @@ export class GoldenFileTest {
    * symlink bazel puts into bazel-bin back into the test_files directory
    * and chopping off the test_files portion.
    */
-  get workspaceRoot(): string {
-    if (!this.tsFiles) {
-      return '';
+  getWorkspaceRoot(): string {
+    if (!this.tsFiles || !this.tsFiles[0]) {
```

Comment:
what about `!this.tsFiles.length` for the second condition? Avoids indexing into an array out of bounds.

Reply:
Done.